integration-tests/test_checkin.py

```diff
@@ -9,8 +16,29 @@
 
 
 def test_ultralight_checkin(insights_client, test_config):
-    """test --checkin
-    Sends nothing but canonical facts to update the host stale and updated timestamp
+    """
+    :id: c662fd5e-0751-45e4-8477-6b0d27f735ac
+    :title: Test verify ultra-light check-in updates timestamps
```

Comment:
This sounds complicated, how about "Lightweight check-in updates staleness timestamps"?